src/common/Locales.cpp

```diff
@@ -89,6 +89,7 @@ std::string Locales::init(std::string i_locale) {
   default_LANGUAGE = Environment::get_variable("LANGUAGE");
 
   setlocale(LC_ALL, "");
+  setlocale(LC_NUMERIC, "C");
```

Comment:
Works around the problem with standard libc functions being locale-dependent, causing a bunch of crashes